packages/bootstrap-vue-next/src/plugins/modalController/index.ts

```
@@ -1,105 +1,216 @@
+          const modal = modals.value.get(_id)
+          if (!modal) return
+          modal.promise.stop?.()
+          if (modal.modelValue) {
+            await new Promise(async (resolve) => {
+              modal.modelValue = false
+              modal['onHidden'] = () => {
+                resolve(undefined)
+              }
+              await nextTick()
+              modal['onUpdate:modelValue']?.(false)
+            })
```

Biome (1.9.4)

[error] Promise executor functions should not be `async`.



(lint/suspicious/noAsyncPromiseExecutor)